app/src/main/java/org/wikipedia/activity/BaseActivity.kt

```diff
@@ -54,6 +62,11 @@ abstract class BaseActivity : AppCompatActivity() {
     private var startTouchY = 0f
     private var startTouchMillis = 0L
     private var touchSlopPx = 0
+    var callback: Callback? = null
+
+    val requestPermissionLauncher = registerForActivityResult(ActivityResultContracts.RequestPermission()) { isGranted: Boolean ->
```

Comment:
Feel like it the `isGranted: Boolean` can be simplified to `isGranted` without the type, or we can directly use the `it`.